server/pbench/bin/pbench-verify-s3-backup-tarballs

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import glob
+import hashlib
+import boto3
+import configtools
+
+options = []
+opts, args = configtools.parse_args(options)
+conf, files = configtools.init(opts)
+
+# configuration file
+endpoint_url = conf.get('aws-s3', 'endpoint_url')
```

Comment:
Can we consider renaming the configuration file section to `[pbench-server-backup]`?